content/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages.md

```diff
@@ -125,11 +125,7 @@ To ensure the repository's URL is correct, replace REPOSITORY with the name of t
 {% data variables.product.prodname_registry %} will match the repository based on the URL, instead of based on the package name. If you store the *package.json* file outside the root directory of your repository, you can use the `directory` field to specify the location where {% data variables.product.prodname_registry %} can find the *package.json* files.
```

Comment:
```suggestion
{% data variables.product.prodname_registry %} will match the repository based on the URL, instead of based on the package name.
```

If users have a mono-repo with packages in different directories, the expanded format with the `directory` field is still valid. I don't think we need to document this somewhat advanced npm feature here. The `directory` field isn't used anywhere by GitHub Packages.

Reply:
it's used by other cli tools like lerna IIRC (which automate a lot of stuff that the npm cli does)